src/pesterTestController.ts

```diff
@@ -473,6 +473,33 @@ export class PesterTestController implements Disposable {
 			await this.powerShellExtensionClient.GetVersionDetails()
 		}
 
+		let pesterModulePath = workspace
+			.getConfiguration('pester')
+			.get<string>('pesterModulePath')
+
+		if (pesterModulePath) {
+			log.debug('A custom Pester module path was specified:', pesterModulePath)
+
+			if (!isAbsolute(pesterModulePath)) {
+				if (workspace.workspaceFolders) {
+					// TODO: Hard-coded to first workspace folder for now,
+					//       but should look through all workspace folders
+					//       for a working path to Pester module
+					const workspaceFolder = workspace.workspaceFolders[0]
+					const workspaceFolderPath = workspaceFolder.uri.fsPath
+					pesterModulePath = join(workspaceFolderPath, pesterModulePath)
+				} else {
+					log.debug(
```

Comment:
Can it get here, is this unnecessary? The extension only runs if there is at least one open project, right?

Reply:
This is fine for now, we'll clean this up as a later multi-workspace PR